test-apps/display-test-app/package.json

```diff
@@ -74,15 +74,13 @@
     "cpx": "^1.5.0",
     "cross-env": "^5.1.4",
     "electron": "^11.1.0",
-    "eslint": "^6.8.0",
```

Comment:
Did you need to remove `eslint` here for this to work?

Reply:
I removed it because its a direct dep of react-scripts and should be included, but youre right its used by the backend so i should explicitly still depend on it. Wasn't sure if i should also bump eslint throughout the monorepo to ^7.11.0.

I did have to set `SKIP_PREFLIGHT_CHECK=true` in the `.env ` file because during compilation it was complaining about the two different versions of eslint. For the frontend this is fine and can be ignored because we are now using the `DISABLE_ESLINT_PLUGIN` flag

Reply:
Ah okay, yes you can go ahead and update the ESLint version.  This update to CRA 4 was actually what we were waiting on before updating the repo.

Reply:
bumped to ^7.11.0